airmozilla/popcorn/views.py

```diff
@@ -113,6 +113,31 @@ def popcorn_data(request):
         }
 
 
+@json_view
+@login_required
+@require_POST
+def save_edit(request):
+    slug = request.POST.get('slug')
+    if not slug:
+        return http.HttpResponseBadRequest('slug')
+    data = request.POST.get('data')
+    if not data:
+        return http.HttpResponseBadRequest('data')
+
```

Comment:
That's one step of validation but `{}` is no more valid than `{'foo': 'bar'}` so it seems kind of pointless to me.